change/@fluentui-react-skeleton-2297a865-9a11-49df-a53b-7ae13309e3d7.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32077&commitId=f1ae5084b2511388414cc02510fde732ac203a39&env=prod&clientType=FLUENTUI)


  <details><summary><b>Card Converged - Interactive</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Skeleton converged</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Skeleton converged.Opaque Skeleton with rectangle - RTL.chromium.png|77| Changed|
|Skeleton converged.Opaque Skeleton with circle - High Contrast.chromium.png|2| Changed|
|Skeleton converged.Opaque Skeleton with rectangle - Dark Mode.chromium.png|144| Changed|
|Skeleton converged.Opaque Skeleton with circle - RTL.chromium.png|9| Changed|
|Skeleton converged.Opaque Skeleton with rectangle - High Contrast.chromium.png|2| Changed|
|Skeleton converged.Opaque Skeleton with rectangle.chromium.png|68| Changed|
|Skeleton converged.Opaque Skeleton with square - Dark Mode.chromium.png|23| Changed|
|Skeleton converged.Opaque Skeleton with circle - Dark Mode.chromium.png|19| Changed|
|Skeleton converged.Translucent Skeleton with rectangle - Dark Mode.chromium.png|12| Changed|
|Skeleton converged.Opaque Skeleton with square.chromium.png|13| Changed|
|Skeleton converged.Opaque Skeleton with square - RTL.chromium.png|13| Changed|
|Skeleton converged.Opaque Skeleton with circle.chromium.png|10| Changed|
|Skeleton converged.Translucent Skeleton with rectangle.chromium.png|11| Changed|
|Skeleton converged.Translucent Skeleton with circle.chromium.png|1| Changed|
|Skeleton converged.Translucent Skeleton with square - Dark Mode.chromium.png|2| Changed|
|Skeleton converged.Translucent Skeleton with square.chromium.png|2| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>